packages/Path-View-Core.package/SClassDiagram.class/instance/dragBottomBy..st

```diff
@@ -0,0 +1,5 @@
+geometry
+dragBottomBy: aValue
```

Comment:
there is nothing which belongs to dragging in this methods, so they could be named like `extendBottomBy: aValue`.